packages/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-responses.ts

```diff
@@ -0,0 +1,61 @@
+import * as z from "zod";
+import {
+  baseResponseSchema,
+  baseErrorResponseSchema,
+  operationOutcomeSchema,
+  externalGatewayPatientSchema,
+  XCPDGatewaySchema,
+} from "../shared";
+
+const patientDiscoveryRespSuccessfulDefaultSchema = baseResponseSchema.extend({
+  patientMatch: z.literal(true),
+  patientResource: z.object({}),
+  xcpdHomeCommunityId: z.string(),
+});
+
+// TO EXTERNAL GATEWAY
+const patientDiscoveryRespToExternalGWSuccessfulSchema =
+  patientDiscoveryRespSuccessfulDefaultSchema.extend({
+    patientMatchDegree: z.number().optional(),
+    externalGatewayPatient: externalGatewayPatientSchema,
+  });
+
+const patientDiscoveryRespToExternalGWFaultSchema = baseErrorResponseSchema.extend({
+  patientMatch: z.literal(false).or(z.literal(null)),
```

Comment:
so this to me looks like patientMatch=False you are categorising a a fault schema, but imo its a successful schema. It just means they didn't have the patient. 
The operationOutcome I am pretty sure is optional when its False. 

I actually think the operation outcome might be optional when patient match is null too